### PR TITLE
Sync with platform rename to p8-platform

### PR DESCRIPTION
**Pull request by spatnynick**
#149 
Pulse-Eight/platform#20

### CODE REVIEW

**Comment by Jalle19** [comment]
@wsnipex do we need to update anything with regards to packaging?

**Comment by wsnipex** [comment]
yes, this need packaging changes as well, see debian dir. Also, libp8platform needs packaging first.
@opdenkamp are you going to update the p8-ppa? or maybe create a new one for the transition period?

**Comment by Jalle19** [comment]
Since the header names have changed I don't think the previously suggested approach (having the Debian package depend on either `libplatform-dev` or `libp8platform-dev`) will work, we need to depend on the newer name explicitly from now on.

As soon as the packages really are available we can update the control file and merge this (assuming it builds).

**Comment by opdenkamp** [comment]
@wsnipex will do

**Comment by Jalle19** [comment]
@opdenkamp any update on the packaging?

**Comment by opdenkamp** [comment]
Next week, still doing holiday stuff, fixing things in the house ;)


-------- Oorspronkelijk bericht --------
Van: Sam Stenvall <notifications@github.com> 
Datum:03-01-2016  14:56  (GMT+01:00) 
Aan: "kodi-pvr/pvr.hts" <pvr.hts@noreply.github.com> 
Cc: Lars Op den Kamp <opdenkamp@gmail.com> 
Onderwerp: Re: [pvr.hts] Sync with platform rename to p8-platform (#154) 

@opdenkamp any update on the packaging?

File: CMakeLists.txt [hunk 1]
```diff
@@ -6,7 +6,7 @@ enable_language(CXX)
 
 find_package(kodi REQUIRED)
 find_package(kodiplatform REQUIRED)
-find_package(platform REQUIRED)
+find_package(p8-platform REQUIRED)
 
 include_directories(${kodiplatform_INCLUDE_DIRS}
                     ${platform_INCLUDE_DIRS}
```
Comment (hudokkow) [inline CMakeLists.txt]: This also needs to be updated, correct?

Comment (opdenkamp) [inline CMakeLists.txt]: aye

**Comment by hudokkow** [comment]
> yes, this need packaging changes as well, see debian dir.

@wsnipex, what packaging changes? I don't see anything mentioning lib(p8-)platform-dev specifically

**Comment by wsnipex** [comment]
right sorry, its used implicitly by libkodiplatform-dev. No change needed then.

**Pull request closed without merging**